prompttools/experiment/llms/compare_responses.py

```diff
@@ -0,0 +1,62 @@
+from langchain import HuggingFaceHub, PromptTemplate, LLMChain
+from prompttools.structures.prompt import Response
+from prompttools.utils import similarity
+from typing import Any, List
+
+
+class MaxLLM:
```

Comment:
You can implement this on the experiment class, rather than creating a separate class for it. It should be one method that gives the top n responses based on a given metric. 

The function signature would look like this 
```
def top_n(self, metric_name: str) -> List
```
where the `metric_name` is for a metric which has already been computed at the `evaluate` step (see https://github.com/hegelai/prompttools/blob/main/prompttools/experiment/experiment.py#L155-L187)

For reference, we already have a `rank` method on the experiment which ranks across multiple rows by aggregating https://github.com/hegelai/prompttools/blob/main/prompttools/experiment/experiment.py#L290-L312. This could be implemented similarly.